core/commands/dht.go

```diff
@@ -581,3 +597,18 @@ PutValue will store the given key value pair in the dht.
 	},
 	Type: notif.QueryEvent{},
 }
+
+func escapeDhtKey(s string) (key.Key, error) {
+	if s[0] != '@' {
+		return key.B58KeyDecode(s), nil
+	}
```

Comment:
ahhh. interesting. that's annoying. yeah base58 sucks for that. 

I still want to make `base4096`, for accent-based url-shortening :)